nemo/collections/nlp/data/language_modeling/megatron/dataset_utils.py

```diff
@@ -439,6 +441,9 @@ def build_train_valid_test_datasets(
 
     if len(data_prefix) == 1:
         return _build_train_valid_test_datasets(
+            cfg,
+            trainer,
+            tokenizer,
             data_prefix[0],
```

Comment:
why there are two `tokenizer`s (line 446 and line 459) in this `_build_train_valid_test_datasets` arguments?

Reply:
Good point! I removed it below but not here. I just fixed it now.